### PR TITLE
Bugfix/numeric text toolbar

### PR DESCRIPTION
**Pull request by hesamhabibih**

### CODE REVIEW

File: components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx [hunk 1]
```diff
@@ -13,11 +13,29 @@ const TextControl = (props: Control) => {
           {props.settings?.LABEL.Data[locale]}
         </label>
       ) : null}
-      <input
-        readOnly
-        className={`appearance-none block w-full bg-white text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white`}
-        placeholder={props?.settings?.PLACEHOLDER.Data[locale] || ''}
-      />
+      {props?.settings?.FIELD_TYPE.Data === 'SINGLE_LINE' ? (
```
Comment (amirkh330) [inline components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx]: please use the types that you made for field type

File: components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx [hunk 2]
```diff
@@ -13,11 +13,29 @@ const TextControl = (props: Control) => {
...
+        <input
+          readOnly
+          className={`appearance-none block w-full bg-white text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white`}
```
Comment (saralabafi) [inline components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx]: you don't have any variable here

File: components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx [hunk 3]
```diff
@@ -13,11 +13,29 @@ const TextControl = (props: Control) => {
...
+          value={
+            props?.settings?.PREDEFINED_VALUE?.Data[locale] != null
+              ? props.settings.PREDEFINED_VALUE.Data[locale]
+              : ''
+          }
+        />
+      ) : (
+        <textarea
+          readOnly
+          className={`appearance-none block w-full bg-white text-gray-700 border rounded py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white`}
```
Comment (saralabafi) [inline components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx]: same here

Comment (hesamhabibih) [reply components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx]: ok

File: components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx (hunk 2 above)
Comment (hesamhabibih) [reply components/Tools/ContentStructureBuilder/Contrrols/TextControl/TextControl.tsx]: ok

**Pull request merged**